src/clients/private/orgs/subscriptions/OrgSubscriptionConnectsPage/ContentFunnels/PrevNextButtons.tsx

```diff
@@ -18,31 +24,46 @@ export const PrevNextButtons = memo(function PrevNextButtons() {
     const clearFormData = useResetRecoilState(createSubscriptionFormData);
     const [teamMembers, setTeamMembers] = useRecoilState(selectedTeamMembersAtom);
     const [currentStep, setStep] = useRecoilState(currentStepAtom);
+    const [invoiceAccountData, setInvoiceAccountData] = useRecoilState(createSubscriptionForInvoiceAccountFormData);
     const {currentConnectingProduct, finishProduct} = useCurrentConnectingProduct();
     const resetFinishedProductMap = useResetRecoilState(finishedProductMapAtom);
     const {refetch} = useWorkspaceSubscriptionCount(orgId);
 
-    //currentConnectingProduct?.id === product.id
-
     const prev = (i: number) => i - 1;
     const next = (i: number) => i + 1;
 
     const createSubscription = () => {
-        subscriptionApi.create(formData).then((res) => {
-            const subscription = res.data;
-            toast.success(`${currentConnectingProduct?.nameKo} 구독을 등록했어요.`);
-            setTeamMembers([]);
-            clearFormData();
-            const nextProduct = finishProduct(subscription.productId);
-
-            if (nextProduct) {
-                setStep(Steps.IsFreeTier);
-            } else {
-                return router.push(OrgMainPageRoute.path(subscription.organizationId)).then(() => {
-                    resetFinishedProductMap();
+        subscriptionApi
+            .create(formData)
+            .then((res) => {
+                const subscription = res.data;
+                Promise.allSettled(
+                    invoiceAccountData.invoiceAccountIds.map((invoiceAccountId) =>
+                        invoiceAccountApi.subscriptionsApi.create(invoiceAccountId, subscription.id),
+                    ),
+                ).then(() => subscription);
+                return subscription;
+            })
```

Comment:
```suggestion
            .then(async (res) => {
                const subscription = res.data;
                await Promise.allSettled(
                    invoiceAccountData.invoiceAccountIds.map((invoiceAccountId) =>
                        invoiceAccountApi.subscriptionsApi.create(invoiceAccountId, subscription.id),
                    ),
                ).then(() => subscription);
                return subscription;
            })
```

정상적인 흐름이라면 async/await이 붙어야 할 것 같아요. (붙이지 않은 상태에서도 동작 자체는 하겠지만)